### ISSUE TITLE
Address color contrast issues in default theme.

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5187]
The following text does not have sufficient colour contrast, as identified in our accessibility audit.

![Selection_139](https://user-images.githubusercontent.com/2306629/67107958-59dcba80-f1c5-11e9-9619-bbd11484e2e3.png)

The default header color needs to be _slightly_ darker to ensure that the search button's contrast is sufficient.

And the background color of the header when viewing a PDF needs to be darker -- probably the same color has the header of the site.

**Comment by NateWr** [comment]
The following PRs address this issue as well as other accessibility issues:  #5188, #5182, #5189, #5181, #5619, and #5184. They also include a fix for #4930.

PRs:
https://github.com/pkp/customBlockManager/pull/66
https://github.com/pkp/pkp-lib/pull/6332
https://github.com/pkp/ui-library/pull/122
https://github.com/pkp/ojs/pull/2901
https://github.com/pkp/omp/pull/875
https://github.com/pkp/ops/pull/76

**Comment by NateWr** [comment]
All merged! These PRs address all of the remaining urgent issues from the accessibility audit of the default theme. :tada: 

@kaitlinnewson and @amandastevens this may effect documentation in a few ways. I have a list I can share that will be covered in the technical release notebook.
